storage/src/test/java/org/apache/kafka/server/log/remote/metadata/storage/TopicBasedRemoteLogMetadataManagerTest.java

```diff
@@ -370,4 +375,58 @@ public void testInitializationFailure() throws IOException, InterruptedException
             Exit.resetExitProcedure();
         }
     }
+
+    @ClusterTest
+    public void testRemoteLogMetadataTopicMinIsr() throws ExecutionException, InterruptedException {
+        // Initialize the manager which will create the __remote_log_metadata topic
+        TopicBasedRemoteLogMetadataManager rlmm = topicBasedRlmm();
+        verifyRemoteLogMetadataTopicMinIsr(rlmm, (short) 2, "default value");
```

Comment:
```suggestion
        verifyRemoteLogMetadataTopicMinIsr(remoteLogMetadataManager, (short) 2, "default value");
```

Reply:
Could you address this minor comment? Why do we want to re-initialize the RLMM?

Reply:
Hi @kamalcph 

The field TopicBasedRemoteLogMetadataManagerTest#remoteLogMetadataManager maybe null if the first running case is this one. So we can using topicBasedRlmm() similar with some other cases to make sure it must be initialized. If it is already initialized. it will return the existed one.

BTW: I remove all others cases and debug with the case if we only use remoteLogMetadataManager directly instead of topicBasedRlmm(). You can see it is null.
<img width="1047" height="277" alt="Image" src="https://github.com/user-attachments/assets/4f9126d9-4bd7-4c0a-b323-7b4a415c0f94" />

thanks a lot!